### DISCUSSION TITLE
3.9.5.81-v6用自带注入工具微信闪退

### DISCUSSION THREAD

**Discussion by cyonjan** [ttttupup/wxhelper#236] [Q&A]
用楼主自带工具注入最新版3.9.5.81-v6，微信闪退，没有任何提示，如何解决。
自从wxhelper.dll升级64位后，用各种工具注入都没有成功过，恳请高手指点。

**Comment by ttttupup** [comment]
可以参考 https://github.com/ttttupup/wxhelper/discussions/65  
个人一直用的这个注入工具，有问题的话，自己调试一下，逆向第一步，不能让个注入劝退了。

**Comment by cyonjan** [comment]
> 可以参考 #65 个人一直用的这个注入工具，有问题的话，自己调试一下，逆向第一步，不能让个注入劝退了。
说的是，还有可能是微信我用的64位的，而且微信升级过程没有清理环境，我都排查下。
